crates/aptos/src/config/mod.rs

```diff
@@ -112,6 +118,47 @@ impl CliCommand<GlobalConfig> for SetGlobalConfig {
     }
 }
 
+/// Show the private key for the given profile
+#[derive(Parser, Debug)]
+pub struct ShowPrivateKey {
+    /// Which profile's private key to show
+    #[clap(long)]
+    profile: String,
+}
+
+#[async_trait]
+impl CliCommand<String> for ShowPrivateKey {
+    fn command_name(&self) -> &'static str {
+        "ShowPrivateKey"
+    }
+
+    async fn execute(self) -> CliTypedResult<String> {
+        let config = CliConfig::load(ConfigSearchMode::CurrentDir)?;
+
+        if let Some(profiles) = &config.profiles {
+            if let Some(profile) = profiles.get(&self.profile.clone()) {
+                if let Some(private_key) = &profile.private_key {
+                    return Ok(format!("0x{}", hex::encode(private_key.to_bytes())));
```

Comment:
@gregnazario is this an opportunity to use the new format? Or at least have an option to use the new format. Or are we not ready for that yet.

Reply:
It's unfortunately not ready, but soon...